.dockerignore

```diff
@@ -1,2 +1,3 @@
 .git
+.travis.yml
```

Comment:
I think we still want this

Reply:
removed it. it was part of the `.dockerignore`, but fine for me ;)